Src/FluentAssertions/Specialized/JsonNodeAssertions.cs

```diff
@@ -13,10 +13,11 @@
 
 namespace FluentAssertions.Specialized;
 
-public class JsonNodeAssertions : ReferenceTypeAssertions<JsonNode, JsonNodeAssertions>
+public class JsonNodeAssertions<T> : ReferenceTypeAssertions<T, JsonNodeAssertions<T>>
```

Comment:
Since you've changed the public API, you also need to run `AcceptApiChanges.ps1` to update the snapshot tests.

Reply:
Done.